recipes/waves/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "waves" %}
+{% set version = "0.6.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lanl/waves/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 62fbb8642b79b39cf369ced0ee7c2638caed4b20b9a9e8a9c4afb42e3f4edecc
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script:
+    - {{ PYTHON }} setup.py install
+    - scons man html-github
+    - {{ PYTHON }} package_documentation.py
+  ignore_run_exports:
+    - python_abi
+
+requirements:
+  build:
+    - python {{ python }}
```

Comment:
I don't think this is necessary? It should be enough to have the one under `host` below? 
```suggestion
```
I also don't know what that construct with the template is even trying to do...

Reply:
You may be right. I'll admit I'm a bit fuzzy on the ``host`` section. I'll re-read the current documentation and try to trim the requirements list to only those items that are strictly necessary.

The ``{{ python }}`` is a hold-over from an earlier recipe where the Python version was a build variant: https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#build-variants. It should not be necessary anymore, but I'll test locally to confirm.